code/modules/antagonists/traitor/README.md

```diff
@@ -27,7 +27,7 @@ Objectives are worth a certain amount of progression points, determined by the c
 
 ### The traitor subsystem
 - `newjoin_progression_coeff` - The coefficient multiplied by the expected progression for new joining traitors to calculate their starting progression, so that they don't start from scratch
-- `progression_scaling_deviance` - The value that the entire system revolves around. This determines how much you deviate by compared to your value against the expected progression. Having a deviance of 20 minutes means that you won't get any progression at all, and if objectives were configured to suit this, you'd have the highest reduction you can possibly get. From no deviance to this value, it scales linearly
+- `progression_scaling_deviance` - The value that the entire system revolves around. This determines how your disadvantages are calculated, if you stray from the expected progression. Having a progression value that is `progression_scaling_deviance` minutes off of the expected progression means that you won't get any progression at all, and if objectives were configured to suit this, you'd have the highest reduction you can possibly get. From the expected progression to this value, it scales linearly and it also works in the opposite direction.
```

Comment:
Nitpicky editor in me, but
`From the expected progression to this value, it scales linearly and it also works in the opposite direction.` -> `It scales linearly from the expected progression to this value, can also work in reverse` (This is kinda vague, I think cause I don't fully understand it. What does works in the opposite direction mean)

Reply:
Ah I see what you mean. `progression_scaling_deviance` is the window in which you get more/less points for being behind/ahead of the target reputation by time

Reply:
yes